Aztec/Classes/Converters/NSAttributedStringToNodes.swift

```diff
@@ -705,7 +775,7 @@ private extension NSAttributedStringToNodes {
     ///
     /// - Returns: Leaf Nodes contained within the specified collection of attributes
     ///
-    func createLeafNodes(from attrString: NSAttributedString) -> [Node] {
+    func createLeafNodes(fromString attrString: NSAttributedString) -> [Node] {
```

Comment:
Any reason why we did this name change?

Adding `fromString` would make sense if there was another method with a similar signature, but the variable is clearly an attributed string given its type, and there's no other method with the same signature.